package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "jquery-htmlclean",
```

Comment:
Might want to publish as `components-jquery-htmlclean` so that if the orignial author of the module wants to take the namespace, they could.